src/Modules/UI/Dnn.PersonaBar.UI/admin/personaBar/scripts/persistent.js

```diff
@@ -3,13 +3,7 @@ define(['jquery'], function ($) {
     return {
         init: function (cfg, sf) {
             var defaultSettings = {
-                period: 'Week',
-                comparativeTerm: '1 w',
-                startDate: '',
-                endDate: '',
-                expandPersonaBar: true,
-                expandTasksPane: true,
-                legends: []
+                expandPersonaBar: true
```

Comment:
What about the other defaults. Are they no longer necessary?

Reply:
no longer needed. most of them are used in "Community Analytics" project, i test with it and it works fine. if we meet problem in future, we should fix in that project but not keep in here.